kinbot/parameters.py

```diff
@@ -194,10 +198,26 @@ def __init__(self, file=None):
             # MESMER specific keywords
             'mesmer_command': 'mesmer',
 
+            # UQ KEYWORDS
+            # UQ off/on (0/1)
+            'uq': 0,                   
+            # Number of mess input files generated for each structure
+            'uq_n': 5,
+            # Max number of mess calculations running at once
+            'uq_max_runs' : 5,
+            # Uncertainty in stationary point energy, +/- 0.5 kcal/mol
+            'stPt_uq': 0.5,
```

Comment:
You could rename this then to `well_uq`, which also applies for products of course, but stPt is misleading, see above.

Reply:
changed in code.